docs/_media/roles/usersRef_net.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="petriflow_schema.xsd">
+<document xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://petriflow.com/petriflow.schema.xsd">
```

Comment:
This URL returns 404. Shouldn't the schema from the Petriflow github repository be referenced instead? https://github.com/netgrif/petriflow

Reply:
No, the url returns the petriflow schema. Please try in Postman, or see "Preview" tab in Inspector or Developer tools.

Reply:
It is better to use domain, hence the github repository may be changed in future, and it always can be mapped to the domain.

Reply:
a GET request with Postman does indeed return the schema